manifests/enterprise/password/manage.pp

```diff
@@ -21,27 +21,31 @@
 # @param secret
 #   The secret used to salt the splunk password.
 #
-# @params service
+# @param service
 #   Name of the Splunk Enterprise service that needs to be restarted after files
 #   are updated, not applicable when running in agent mode.
 #
-# @params mode
+# @param mode
 #   The class is designed to work in two ways, as a helper that is called by
 #   Class[splunk::enterprise::config] or leveraged independently from with in a
 #   Bolt Plan. The value defaults to "bolt" implicitly assuming that anytime it
 #   is used outside of Class[splunk::enterprise::config], it is being used by
 #   Bolt
 #
 class splunk::enterprise::password::manage(
-  Boolean $manage_password                   = $splunk::params::manage_password,
-  Stdlib::Absolutepath $password_config_file = $splunk::params::forwarder_password_config_file,
-  String[1] $password_content                = $splunk::params::password_content,
-  Stdlib::Absolutepath $secret_file          = $splunk::params::forwarder_secret_file,
-  String[1] $secret                          = $splunk::params::secret,
-  String[1] $splunk_user                     = $splunk::params::splunk_user,
-  String[1] $service                         = $splunk::params::enterprise_service,
+  Boolean $manage_password                   = lookup($splunk::enterprise::manage_password),
```

Comment:
Why did you add all the `lookup()` calls?

Reply:
After thinking about it, that was silly, and I'm going to remove them